pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -78,6 +78,13 @@ in
         [ darwin.apple_sdk.frameworks.CoreServices ];
   };
 
+  jekyll = attrs: {
+    postInstall = ''
+      installPath=$(cat $out/nix-support/gem-meta/install-path)
+      sed -i $installPath/lib/jekyll/commands/new.rb -e 's@Exec.run("bundle", "install"@Exec.run("nix-shell", "-p", "bundler", "--pure", "--run", "${bundler}/bin/bundle install"@'
```

Comment:
@league I think this exec should be disabled altogether. Nothing that user can specify in that gemfile will propagate to nix'ed jekyll. User would have to add stuff to override pkgs/applications/misc/jekyll/Gemfile . What do you think?